hooks/providers/use-provider-reviews.ts

```
@@ -0,0 +1,51 @@
+
+/**
+ * Hook to fetch reviews for the current provider 
+ * @param limit Number of reviews to fetch (default: 5)
+ * @returns Query result containing provider reviews, loading state, and errors
+ */
+export function useProviderReviews(limit: number = 5) {
+  const getApiClient = useApiClient();
+  const { isSignedIn } = useAuth();
+
+  return useQuery<Review[]>({
+    queryKey: [...queryKeys.reviews.byProvider('me' as any), limit],
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)